fizbuz/__init__.py

```diff
@@ -0,0 +1,2 @@
+__doc_url__ = "https://readthedocs.org/projects/ipython/downloads/htmlzip/stable/"
+__version__ = "1.2.3"
```

Comment:
an aside, but I really like how we lucked out and the `__doc_url__` and `__version__` have the same length. It makes my brain happy 😸

Reply:
the little things.